src/detect-transform-compress-whitespace.c

```diff
@@ -43,7 +43,8 @@ void DetectTransformCompressWhitespaceRegister(void)
 {
     sigmatch_table[DETECT_TRANSFORM_COMPRESS_WHITESPACE].name = "compress_whitespace";
     sigmatch_table[DETECT_TRANSFORM_COMPRESS_WHITESPACE].desc =
-        "modify buffer to strip whitespace before inspection";
+        "modify buffer to commpress consecutive whitespaces "
```

Comment:
nit:s/consecutive whitespaces/consecutive whitespace characters/

Reply:
ok thanks for the English review